common/user-behaviors.js

```diff
@@ -334,6 +334,62 @@ export const downloadZip = async (file) => {
   }
 };
 
+const _nativeDownload = ({ url, onError }) => {
+  const iframe = document.createElement("iframe");
+  iframe.style.display = "none";
+  iframe.src = url;
+
+  const ERROR_MESSAGE = "SLATE_DOWNLOAD_ERROR";
+  const hanldeIframeErrors = (e) => {
```

Comment:
seems to be a misspelling here of "handle"

Reply:
nice solve for the error detection though!

Reply:
you're right @martinalong! I'll fix it

Reply:
don't worry though, i'll make the fix and just merge it